### PR TITLE
[1ES] Add code mirror

### PR DESCRIPTION
**Pull request by liliankasem**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: build/pipelines/templates/code-mirror.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+trigger:
+  branches:
+    include:
+    # Keep this set limited as appropriate (don't mirror individual user branches).
+    - main
+    - release/v1.x
```
Comment (jviau) [inline build/pipelines/templates/code-mirror.yml:6]: @fabiocav did you add rule set to this repo to protect release branches? If so, we do have the option of using `release/*` here.

**Review by jviau — APPROVED** [review]

File: build/pipelines/templates/code-mirror.yml (hunk 1 above)
Comment (jviau) [reply build/pipelines/templates/code-mirror.yml:6]: But can always make that glob change later.

File: build/pipelines/templates/code-mirror.yml
Comment (jviau) [inline build/pipelines/templates/code-mirror.yml:1]: _technically_ this is not a template, but a pipeline itself.

Comment (liliankasem) [reply build/pipelines/templates/code-mirror.yml:1]: Ah yup, totally missed that - will move!

**Review by fabiocav — APPROVED** [review]

File: build/pipelines/templates/code-mirror.yml (hunk 1 above)
Comment (fabiocav) [reply build/pipelines/templates/code-mirror.yml]: The change is in place. So we can update the pattern.

**Pull request merged**